search/app.js

```diff
@@ -153,13 +161,17 @@ const getCollection = async (event, parsedPath) => {
 const getGranules = async (event, parsedPath) => {
   console.log(`getGranules ${JSON.stringify(parsedPath, null, 2)}`);
   const conceptId = parsedPath[1];
-  const params = _.merge(
-    adaptParams(wfsParamsToCmrParamsMap, event.queryStringParameters),
-    { collection_concept_id: conceptId }
-  );
+  // TODO: remove lodash
+  const params = Object.assign(adaptParams(wfsParamsToCmrParamsMap, event.queryStringParameters), { collection_concept_id: conceptId })
```

Comment:
Same as above, except the only thing you need to add here is the first argument of `{}`

Reply:
Cool, this is now done.